src/styles/tailwind.css

```diff
@@ -34,13 +34,13 @@
   font-size: 14px;
 }
 @property --discord-button-angle {
-  syntax: "<angle>";
```

Comment:
since there are no changes in this file can you revert this file back to how it is on production

Reply:
Done aswell sir👍